schemas/com.snowplowanalytics.snowplow/anon_ip/jsonschema/1-0-1

```diff
@@ -0,0 +1,44 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an IP anonymization enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "anon_ip",
+		"format": "jsonschema",
+		"version": "1-0-1"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+			"type": "string"
+		},
+		"name": {
+			"type": "string"
+		},
+		"enabled": {
+			"type": "boolean"
+		},
+		"parameters": {
+			"type": "object",
+			"properties": {
+				"anonOctets": {
+					"description": "Number of IPv4 octets to anonymize",
+					"type": "number",
+					"minimum": 1,
+					"maximum": 4
+				},
+				"anonSegments": {
+					"description": "Number of IPv6 segments to anonymize",
+					"type": "integer",
```

Comment:
Is there a reason why `anonOctets` is a `number` and `anonSegments` an `integer`? I presume they should be both `integer` but we leave `anonOctets` as `number` for backward compability, but would like to be sure

Reply:
Is there an answer to this @chuwy ?

Reply:
Yup. For some reasons we have `number` for `anonOctets` in 1-0-0, which is obviously too permissive. We added `anonSegments` as `integer` in 1-0-1, which is better, but cannot change `anonOctets` to `integer` because it will be REVISION then.